src/client/dfs/dfs.c

```diff
@@ -7939,3 +7939,176 @@ dfs_readdir_with_filter(dfs_t *dfs, dfs_obj_t *obj, dfs_pipeline_t *dpipe, daos_
 	D_FREE(buf_keys);
 	return rc;
 }
+
+struct dfs_scan_args {
+	time_t		start_time;
+	time_t		print_time;
+	uint64_t	cur_depth;
+	uint64_t	max_depth;
+	uint64_t	num_files;
+	uint64_t	num_dirs;
+	uint64_t	num_symlinks;
+	uint64_t	total_bytes;
+	uint64_t	largest_file;
+	uint64_t	largest_dir;
+	uint64_t	num_scanned;
+};
+
+static int
+scan_cb(dfs_t *dfs, dfs_obj_t *parent, const char name[], void *args)
+{
+	struct dfs_scan_args	*scan_args = (struct dfs_scan_args *)args;
+	dfs_obj_t		*obj;
+	daos_obj_id_t		oid;
```

Comment:
build failed because you also need to remove that variable since it's unused anymore.